pages/create-account.js

```diff
@@ -0,0 +1,138 @@
+import Head from 'next/head';
+import React, { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Account() {
+  const [inputErrorMessages, setErrorMessages] = useState({
+    firstName: '',
+    lastName: '',
+    username: '',
+    password: '',
+    confirmPassword: '',
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const inputs = {};
+    const formData = new FormData(e.target);
+    formData.forEach((value, key) => (inputs[key] = value));
+    
+    // Validation criteria for each field
+    for (const input in inputs) {
+      let errorMessage;
+      switch (input) {
+        case 'firstName':
+          errorMessage = inputs[input].length == 0 ?
```

Comment:
Use '===' instead of '==' because we know length will always return an `int` so it'll pass a type check with 0.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality